app.py

```diff
@@ -47,6 +48,8 @@ def get(self):
     ('/flickr/oauth_callback', flickr.CallbackHandler.to('/')),
     ('/googleplus/start', googleplus.StartHandler.to('/googleplus/oauth_callback')),
     ('/googleplus/oauth_callback', googleplus.CallbackHandler.to('/')),
+    ('/indieauth/start', indieauth.StartHandler.to('/indieauth/oauth_callback')),
+    ('/indieauth/oauth_callback', indieauth.CallbackHandler.to('/')),
```

Comment:
sure it is! maybe [citation needed], but IndieAuth implementations conform to the OAuth 2 spec.